build/functions.sh

```diff
@@ -0,0 +1,86 @@
+#!/usr/bin/env bash
+
+start_time="$(date -u +%s)"
+export start_time
+
+export CGO_ENABLED=0
+export GOOS=linux
+export GOARCH=amd64
+
+export GREEN='\033[0;32m'
+export NC='\033[0m' # No Color
+
+mkdir -p "${BIN_DIR}"
+
+function appToSrc {
+    case "$1" in
+        admission-webhook)      echo "./cmd/admission-webhook" ;;
```

Comment:
@nickolaev I have the same base reaction you do here... its another place to maintain things.  That said, we have simply relocated this information from Dockerfiles to this script.  I'd like to see if we can think of a better way though.

Reply:
make file approach is also not good in my view, since it still require a lot of stuff to maintain.
This Pr is more like check if cross compile is working fine for us.

So probable we could go with generic make file approach for any of components, or we could choose some from this list: http://go-lang.cat-v.org/dev-utils